llvm/test/CodeGen/AArch64/ptrauth-elf-globals-pic.ll

```diff
@@ -0,0 +1,23 @@
+; RUN: llc -mtriple=arm64 -global-isel=0 -fast-isel=0         -relocation-model=pic -o - %s -mcpu=cyclone -mattr=+pauth | FileCheck %s
```

Comment:
It might not be necessary to have basic-pic elf-globals-pic.ll extern-weak.ll. You can combine them into one file.

Reply:
I'll try (as a separate follow-up patch) to combine such tests in two files: one for ptrauth-*.ll tests and one for corresponding non-pauth tests. Thanks for suggestion.